src/control/server/ioserver/config_test.go

```diff
@@ -294,22 +294,24 @@ func TestIOServer_ConfigValidation(t *testing.T) {
 
 func TestConfigToCmdVals(t *testing.T) {
 	var (
-		mountPoint     = "/mnt/test"
-		provider       = "test+foo"
-		interfaceName  = "qib0"
-		modules        = "foo,bar,baz"
-		systemName     = "test-system"
-		socketDir      = "/var/run/foo"
-		logMask        = "LOG_MASK_VALUE"
-		logFile        = "/path/to/log"
-		cfgPath        = "/path/to/nvme.conf"
-		shmId          = 42
-		interfacePort  = 20
-		targetCount    = 4
-		helperCount    = 1
-		serviceCore    = 8
-		index          = 2
-		pinnedNumaNode = uint(1)
+		mountPoint         = "/mnt/test"
+		provider           = "test+foo"
+		interfaceName      = "qib0"
+		modules            = "foo,bar,baz"
+		systemName         = "test-system"
+		socketDir          = "/var/run/foo"
+		logMask            = "LOG_MASK_VALUE"
+		logFile            = "/path/to/log"
+		cfgPath            = "/path/to/nvme.conf"
+		shmId              = 42
+		interfacePort      = 20
+		targetCount        = 4
+		helperCount        = 1
+		serviceCore        = 8
+		index              = 2
+		pinnedNumaNode     = uint(1)
+		crt_ctx_share_addr = uint32(1)
+		crt_timeout        = uint32(30)
```

Comment:
This being a Go test, I think these names should be camelCase.

Reply:
Done.